File: Serene/Serene.Script/Northwind/Shipper/ShipperFormatter.cs

namespace Serene.Northwind
{
    using Serenity;
    using System.Runtime.CompilerServices;

    [Imported]
    public class ShipperFormatter : ISlickFormatter
    {
        public string Format(SlickFormatterContext ctx)
        {
            return "<span class='shipper-symbol shipper-" + 
                ((ctx.Value as string) ?? "").Replace(" ", "") + 
                "'>" + Q.HtmlEncode(ctx.Value) +
                "</span>";
        }
    }
}

File: Serene/Serene.Core/Modules/Northwind/Shipper/ShipperFormatter.ts
namespace Serene.Northwind {

    @Serenity.Decorators.registerFormatter()
    export class ShipperFormatter implements Slick.Formatter {
        format(ctx: Slick.FormatterContext) {
            return "<span class='shipper-symbol shipper-" +
                Q.replaceAll((ctx.value || '').toString(), ' ', '') +
                "'>" + Q.htmlEncode(ctx.value) + '</span>';
        }
    }
}